src/clog/core.clj

```diff
@@ -1,20 +1,24 @@
 (ns clog.core
   (use [ring.adapter.jetty]
-       [hiccup.core])
-  (:require [hiccup.page :as page]))
+       [hiccup.core]
+       [ring.middleware.params]
+       [clog.presenter]
+       [clog.blogs]))
 
-(defn- page []
-  (page/html5
-    [:head
-      [:title "Clog"]]
-    [:body
-      [:div "Hello and welcome to Clog"]]))
-
-(defn -main
-  [& args]
-  (println "Hello World"))
-
-(defn handle [request]
+(defn- response [page]
   {:status 200
    :headers {}
-   :body (page)})
+   :body page})
+
+(defn- save-request [parameters]
+  (save-blog (first parameters)))
+
+(defn- handle [request]
+  (let [page (first (:query-params request))
```

Comment:
I would use destructuring here rather than a let block

This could be wrong but I think something like
```
(defn- handle [{[page & params] :query-params}]
 )
```
would work